Packager/Factories/CodingHistory/DatCodingHistoryGenerator.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Packager.Extensions;
+using Packager.Models.FileModels;
+using Packager.Models.PodMetadataModels;
+using System.Collections;
+using System.Collections.Generic;
+using System.Data;
+
+namespace Packager.Factories.CodingHistory
+{
+
+    public class DatCodingHistoryGenerator : AbstractCodingHistoryGenerator
+    {
+        private const string Line1Format = "A={0},M={1},T={2},";
+        private const string Line2Format = "A=PCM,F={0},W={1},M={2},T={3};A/D,";
+        private const string Line3Format = "A=PCM,F={0},W={1},M={2},T=Lynx AES16;DIO";
+
+        private const string ThirtyTwoK = "32k";
+        private const string FortyFourOneK = "44.1k";
+        private const string FortyEightK = "48k";
+        private const string NinetySixK = "96k";
+        private const string TwentyFourBit = "24";
+        private const string SixteenBit = "16";
+        private IDictionary<string, string> SampleRatesToText = new Dictionary<string, string>()
```

Comment:
here, you can define the dictionary as having a case-insensitive comparer: ` new Dictionary<string, string>(StringComparer.InvariantCultureIgnoreCase)`. Also, you can make this field `readonly`

Reply:
Resolved https://github.com/IUMDPI/IUMediaHelperApps/pull/29/commits/9257104f1a655b089a7c7d44db46738ccada92f9